### ISSUE TITLE
Should configure git cli on start

### ISSUE THREAD

**Issue by btkostner** [WeblateOrg/docker#221]
The docker container should run the following commands when started to configure git:

```
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
```

**Comment by nijel** [comment]
The committer is configured per repo by Weblate, why would you need this?

**Comment by btkostner** [comment]
@nijel When using the weblate "repository maintenance" page, I get a server 500 error pushing to remote repository. Looking in the logs I get the normal unconfigured git client error telling me to run `git config --global user.email "you@example.com`. There is no `/app/data/home/.gitconfig` created. So I configured it in the docker container and everything seemed to work after.

I _did not_ run the `updategit --all` command after this fix was released, so that could be the issue. https://github.com/WeblateOrg/weblate/issues/2714
